src/augmint/rates.js

```diff
@@ -11,9 +11,13 @@ const AugmintError = require("../augmint/augmint.error.js");
 let dayAdjust = 0;
 let ethUsdHist = [];
 
+rates.accelerate = false;// every 50 days
+rates.refreshPeriod = 50;
```

Comment:
These params don't belong to rates. Store it in simulation just like other sim params